timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -893,6 +893,113 @@ def evaluate(
             logger.info(json.dumps(scores_dict, indent=4))
         return scores_dict
 
+    def feature_importance(
+        self,
+        data: Union[TimeSeriesDataFrame, pd.DataFrame, Path, str],
+        model: Optional[str] = None,
+        metric: Optional[Union[str, TimeSeriesScorer]] = None,
+        features: Optional[List[str]] = None,
+        time_limit: Optional[float] = None,
+        method: Literal["naive", "permutation"] = "permutation",
+        subsample_size: int = 50,
+        num_iterations: Optional[int] = None,
+    ):
```

Comment:
Should we fix the seed here as well for reproducibility (both when selecting `subsample_size` items & permutations)?

Maybe set seed to `123` by default, and then for each of `num_iterations` increment the seed by 1 to ensure reproducible results.

Reply:
Why does the seed need to be incremented (and reset) for each iteration?

Reply:
Never mind, just fixing the seed should be fine.